spring-cloud-dataflow-build/spring-cloud-dataflow-build-dependencies/pom.xml

```diff
@@ -87,13 +88,6 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
-			<dependency>
```

Comment:
Since our parent is SBD, having the direct import of SBD does nothing other than removes the ability to do easy version overrides such as https://github.com/spring-cloud/spring-cloud-dataflow/pull/5595/files#diff-7059f95906ccc5bf99ecd1a8702cabced0dc3f24f787c8d32281c624e4f33943R42.